### DISCUSSION TITLE
关于markdown增强功能，是否直接支持markdown-it插件形式引入？

### DISCUSSION THREAD

**Discussion by daxiongYang** [vuepress-theme-hope/vuepress-theme-hope#1919] [Q&A] [answered]
您好，看到官网文档中的markdown增强功能，实在是太棒了！请问是否支持markdown-it插件形式引入？

**Comment by Mister-Hope** [comment, marked as answer]
这个插件是完全 tree-shaking 的，node 端只会根据你启用的功能注入 markdown-it 插件，client 端是根据你的配置动态生成入口文件。所有你没有启用的功能都不会注入。所以如果你想要使用 vuepress，你应该无需关注内部实现。

如果你单纯对某些markdown 插件比较好奇想要抽离在其他地方使用，你可以查看源码，这些插件应该都进行了导出。比如你想使用 imageMark，你可以通过 


```ts
import { imageMark } from "vuepress-plugin-md-enhance";

// ...
md.use(imageMark, { /* your options */});

```

进行使用

**Reply by daxiongYang** [reply]
好的，再次感谢您的高效回复。非常感谢！真棒
